### DISCUSSION TITLE
記号を含めて記事検索したい

### DISCUSSION THREAD

**Discussion by is0263rf** [increments/qiita-discussions#485] [要望・アイデア・不具合]
プログラミング言語のC#やF#に関する記事を検索したいと考えて「C#」または「F#」を検索欄に入力した場合、#を取り除いた「C」や「F」を含む記事が検索結果に表示されます。「tag:C#」は上手く検索できているようですが、「body:C#」は「C#」ではなく「C」を含む記事が検索結果に表示されます。

Microsoftが提供している開発環境の「.NET」を検索する場合に関しても同様で「.」が取り除かれた「NET」を含む記事が検索に表示されます。

ピリオドやシャープを含めて記事検索する事は出来ないでしょうか?

**Comment by wataru86** [comment]
@is0263rf
ご意見ありがとうございます！
現状の検索システムの都合上、body に対して記号で検索できない仕様になっています。
他の開発優先度との兼ね合いですぐの対応は難しい状況ですが、
今後の検索機能の改修を行う際に、検討させていただきます 🙇

**Comment by tomoasleep** [comment]
@is0263rf 
大変お待たせしました :bow:

検索機能の改修を行い、「C#」や「.NET」などの記号を含めたキーワードの場合も、お目当ての記事が探しやすくなりました！
ぜひご活用ください :sparkles:
https://qiita.com/release-notes/2025-07-04

**Reply by is0263rf** [reply]
ご対応ありがとうございます。今後もQiitaを利用させていただきます。
